### PR TITLE
newer Fedoras use python2-packagename

### PR DESCRIPTION
**Pull request by marcusmueller**

### CODE REVIEW

File: twisted.lwr [hunk 1]
```diff
@@ -21,7 +21,7 @@ category: baseline
 depends: null
 satisfy:
   deb: python-twisted || python-twisted-core
-  rpm: python-twisted || python-twisted-core
+  rpm: python2-twised || python-twisted || python-twisted-core
```
Comment (mbr0wn) [inline twisted.lwr]: typo

**Comment by marcusmueller** [comment]
dammit

**Comment by marcusmueller** [comment]
sorry, my bad. fixed.

**Comment by mbr0wn** [comment]
Looks like it's not fixed...?

**Comment by marcusmueller** [comment]
huh, I --amended that commit and force-pushed it; that's why the commit seems to be /after/ your change request. Anyway, the commit is the old, broken again, so, I'll do the change in a separate commit and let you squash-merge it.

**Comment by marcusmueller** [comment]
better now?

**Pull request merged**